compiler/src/dotty/tools/dotc/util/Signatures.scala

```diff
@@ -48,31 +49,40 @@ object Signatures {
    */
   def callInfo(path: List[tpd.Tree], span: Span)(using Context): (Int, Int, List[SingleDenotation]) =
     path match {
+      case UnApply(fun, _, patterns) :: _ =>
+        callInfo(span, patterns, fun, 0)
```

Comment:
Is this correct for an unapply with multiple parameter lists? This can happen when it has using parameters.

Reply:
Some examples: https://github.com/lampepfl/dotty/blob/a77766535c90d5f1b0a4a0c5238a1456254e0039/tests/pos/splice-pat.scala

Reply:
Ach, thanks! I will take a look at that tomorrow.